docs/build/build-client-side.md

```diff
@@ -0,0 +1,187 @@
+and which prefixes are available.
+
+:::
+
+## SDKS & Libraries
+
+If one aims to develop a **dApp** (Decentralized App) or **uApp** (Unstoppable App), the Polkadot
+ecosystem contains various SDKs to tap into the relay chain and parachains. There are several
+languages already supported -
+[see the tooling page for a detailed overview of different SDKs and libraries that are available.](./build-tools-index.md)
+
+## Frameworks & Toolkits
+
+For front-end applications, several options exist for interfacing with Substrate-based chains
+(parachains, relay chains, etc.) and smart contracts. These often will interact with the RPC of a
+Substrate node:
+
+<!-- prettier-ignore -->
+<Tabs groupId="clients" values={[ {label: 'Polkadot.js', value: 'pjs'}, { label: 'Polkadot API (under development)', value: 'papi'}, {label: 'Subxt', value: 'subxt'}, {label: 'React Hooks for ink!', value: 'useink'}, {label: 'ink!athon Boilerplate', value: 'inkathon'}, { label: 'Polkadot Cloud', value: 'pcloud'} ]}>
+
+<TabItem value="pjs"> Promise and RxJS APIs around Polkadot and Substrate-based chains via RPC
+calls. It is dynamically generated based on what the Substrate runtime provides regarding metadata.
+Full documentation & examples
+available&nbsp;<a href="https://polkadot.js.org/docs" target="_blank">here</a>. </TabItem>
+
+<TabItem value="papi"> The Polkadot API (formerly known as "CAPI") is currently under development,
+but will serve as a replacement for Polkadot JS. While still under construction, you may view the
+current progress on GitHub
+&nbsp;<a href="https://github.com/paritytech/polkadot-api/" target="_blank">here</a>. </TabItem>
+
+<TabItem value="subxt"> Query and submit extrinsics (transactions) to a Substrate node via RPC using
+Rust. Also referred to as Rust Parity. Full documentation & examples
+available&nbsp;<a href="https://github.com/paritytech/subxt" target="_blank">here</a>. </TabItem>
+
+<TabItem value="useink"> React hooks library for ink! smart contracts that abstract the
+functionality of polkadot.js. Full documentation & examples
+available&nbsp;<a href="https://use.ink" target="_blank">here</a>. </TabItem>
+
+<TabItem value="inkathon">ink!athon is a starter kit for full-stack dApp development with ink! smart
+contracts and a React-based frontend in one place. With convenient helper scripts and a
+pre-configured project setup, you can quickly scaffold any dApp. Live example & full documentation
+available&nbsp;<a href="https://inkathon.xyz" target="_blank">here</a>. </TabItem>
+
+<TabItem value="pcloud">Polkadot Cloud hosts a library of assets, ranging from data sources,
+graphical elements, to fully functional components, for app developers to plug and play into their
+codebases. &nbsp;<a href="https://polkadot.cloud/" target="_blank">Learn more here</a>. </TabItem>
+
+</Tabs>
+
+## Oracle Options for Polkadot
+
+In the blockchain context, an _oracle_ is a way to bring real-world data onto the blockchain so that
```

Comment:
there inbound and outbound oracles. Here you mention inbound only

Reply:
Do we have any good examples of outbound oracle solutions in polkadot atm? 

Isn't an outbound oracle just exporting various bits of data, which events kind of do already in substrate iirc